spec/controllers/catalog_controller_spec.rb

```diff
@@ -28,4 +28,29 @@
       expect(facets).to contain_exactly(*expected_facets)
     end
   end
+
+  describe 'search fields' do
+    let(:search_fields) { controller.blacklight_config.search_fields.keys }
+
+    let(:expected_search_fields) do
+      ['all_fields',
+       'contributor',
+       'creator',
+       'title',
+       'description',
+       'publisher',
+       'date_created',
+       'subject',
+       'language',
+       'resource_type',
+       'format',
+       'identifier',
+       'based_near',
+       'keyword',
+       'depositor',
+       'rights']
+    end
+
```

Comment:
Is this the out-of-box search behavior.  I'm not sure I would expect or want to have searches default against all of these attributes.  Some feel much more appropriate as facets than as search - eg. rights - searching for "copyright" doesn't really produce meaningful results since I get "Copyright not evaluated", "In Copyright", and "No Copyright".  Language similarly feels like something that should be faceted, not searched (at least not by default).

Reply:
Yeah, this is the default and I assumed we would address removing fields that shouldn't be searchable when adding other fields.

Reply:
Ok, then we can defer this until later and this PR is good for now.